lib/Button/Button.less

```diff
@@ -1,4 +1,15 @@
 /* Button */
+
+@-webkit-keyframes moonButtonBulge {
+	0%, 100% { transform: translateZ(0); }
```

Comment:
I didn't test on the TV tonight but on my Mac, the translateZ wasn't necessary here to maintain a compositing layer during and after the animation.